nntrainer/src/neuralnet.cpp

```diff
@@ -948,44 +1007,43 @@ int NeuralNetwork::addLayer(std::shared_ptr<Layer> layer) {
   return status;
 }
 
-std::shared_ptr<Layer>
-NeuralNetwork::_make_act_layer(std::shared_ptr<Layer> layer) {
-  ActiType act = layer->getActivationType();
-  if (act != ACT_UNKNOWN) {
-    if (layers.back()->getType() == LAYER_ACTIVATION) {
-      /* possible throw here? */
-      ml_logw("Activation defined right after activation");
-    }
+std::shared_ptr<Layer> NeuralNetwork::_make_act_layer(ActiType act) {
+  if (layers.back()->getType() == LAYER_ACTIVATION) {
+    /** User defined activation layer. Do not add another activation layer after this */
+    ml_loge("Error: double activation layers.");
```

Comment:
Could you talk to @zhoonit about it? He has intention to have warning instead of error.

Reply:
NBD, I think this can easily be fixed it it becomes a requirement :)

Reply:
In my understanding, this situation only happens due to internal bug as user cannot request for a separate LAYER_ACTIVATION as of now. So, I changed this to error.